widgetssdk/src/main/java/com/glia/widgets/callvisualizer/CallVisualizerSupportActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.glia.widgets.callvisualizer
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.glia.widgets.databinding.CallVisualizerSupportActivityBinding
+
+class CallVisualizerSupportActivity : AppCompatActivity() {
+
+    private lateinit var binding: CallVisualizerSupportActivityBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = CallVisualizerSupportActivityBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+    }
+}
```

Comment:
I think it is possible to have just empty activity without setContent etc
```suggestion
import androidx.appcompat.app.AppCompatActivity

class CallVisualizerSupportActivity : AppCompatActivity()
```

Reply:
true, see previous comment